src/Build/BackEnd/Components/ProjectCache/ProjectCacheService.cs

```diff
@@ -124,20 +118,17 @@ private async Task BeginBuildAsync(ProjectCacheDescriptor? vsWorkaroundOverrideD
                         new DefaultMSBuildFileSystem(),
                         projectDescriptor.ProjectGraph,
                         projectDescriptor.EntryPoints),
-                    logger,
+                    pluginLogger,
                     _cancellationToken);
 
-                timer.Stop();
-                logger.LogMessage($"Finished initializing project cache plugin in {timer.Elapsed.TotalMilliseconds} ms", MessageImportance.Low);
```

Comment:
For these sorts of timing things can you create ETW events? Then we get timing "for free" and in the usual way. https://github.com/dotnet/msbuild/blob/main/src/Framework/MSBuildEventSource.cs

Reply:
This change removes the timers, but I'll note this for the future when I continue to iterate on the logging for this stuff

Reply:
Actually, these are super easy to apply. I'll add them in this PR to replace what I'm removing.